### PR TITLE
Fix HBase init error when Kerberos is enabled

### PR DESCRIPTION
**Pull request by tmljob**
This PR fix 2 errors when  Kerberos is enabled，as follws:
1、The script init-store.sh excludes /jre/lib/ext path；
2、The  hbase-site.xml can‘t be readed when HBase create configuration.

### CODE REVIEW

File: hugegraph-dist/src/assembly/static/bin/init-store.sh [hunk 1]
```diff
@@ -30,7 +30,7 @@ cd $TOP
 
 echo "Initializing HugeGraph Store..."
 
-$JAVA -cp $LIB/hugegraph-dist-*.jar -Djava.ext.dirs=$LIB:$PLUGINS \
+$JAVA -cp $LIB/hugegraph-dist-*.jar -Djava.ext.dirs=$LIB:$PLUGINS:$JAVA_HOME/jre/lib/ext \
```
Comment (javeme) [inline hugegraph-dist/src/assembly/static/bin/init-store.sh:33]: add $EXT var for "$JAVA_HOME/jre/lib/ext" and set -Djava.ext.dirs=$EXT:$LIB:$PLUGINS

File: hugegraph-hbase/src/main/java/com/baidu/hugegraph/backend/store/hbase/HbaseSessions.java [hunk 2]
```diff
@@ -135,6 +136,9 @@ public synchronized void open() throws IOException {
         hConfig.setInt("hbase.hconnection.threads.max",
                        config.get(HbaseOptions.HBASE_THREADS_MAX));
 
+        String hbaseSite = config.get(HbaseOptions.HBASE_HBASE_SITE);
```
Comment (javeme) [inline hugegraph-hbase/src/main/java/com/baidu/hugegraph/backend/store/hbase/HbaseSessions.java:139]: can it be null? don't addResource if hbaseSite is null

**Review by javeme — CHANGES_REQUESTED** [review]

**Comment by javeme** [comment]
please rebase master: rebase -i master and remove 6701022 and 45cf281 commits

File: hugegraph-hbase/src/main/java/com/baidu/hugegraph/backend/store/hbase/HbaseSessions.java (hunk 2 above)
Comment (tmljob) [reply hugegraph-hbase/src/main/java/com/baidu/hugegraph/backend/store/hbase/HbaseSessions.java:139]: This config item's default value is /etc/hbase/conf/hbase-site.xml.If not setting,program will get default value.

**Comment by tmljob** [comment]
> 45cf281

How to rebase and remove? I execute "git rebase -i master" in brach fix-kerberos. It report displays the following output，and then what to do?
```
pick 51d2a6ae Fix bug: hbase init error when kerberos is enabled fix #1292

# Rebase 45cf281e..51d2a6ae onto 45cf281e (1 command)
#
# Commands:
# p, pick <commit> = use commit
# r, reword <commit> = use commit, but edit the commit message
# e, edit <commit> = use commit, but stop for amending
# s, squash <commit> = use commit, but meld into previous commit
# f, fixup <commit> = like "squash", but discard this commit's log message
# x, exec <command> = run command (the rest of the line) using shell
# b, break = stop here (continue rebase later with 'git rebase --continue')
# d, drop <commit> = remove commit
# l, label <label> = label current HEAD with a name
# t, reset <label> = reset HEAD to a label
# m, merge [-C <commit> | -c <commit>] <label> [# <oneline>]
# .       create a merge commit using the original merge commit's
# .       message (or the oneline, if no original merge commit was
# .       specified). Use -c <commit> to reword the commit message.
#
# These lines can be re-ordered; they are executed from top to bottom.
#

```

**Comment by tmljob** [comment]
> please rebase master: `rebase -i master` and remove `6701022` and `45cf281` commits

我通过 git rebase --onto 6701022^ 6701022 来删除这个提交，然后强制push，感觉搞乱掉了呀。 

[Git彻底删除一次提交](https://blog.csdn.net/qwe25878/article/details/89229187?utm_medium=distribute.pc_relevant.none-task-blog-BlogCommendFromBaidu-2.control&depth_1-utm_source=distribute.pc_relevant.none-task-blog-BlogCommendFromBaidu-2.control)

File: hugegraph-hbase/src/main/java/com/baidu/hugegraph/backend/store/hbase/HbaseSessions.java (hunk 2 above)
Comment (javeme) [reply hugegraph-hbase/src/main/java/com/baidu/hugegraph/backend/store/hbase/HbaseSessions.java:139]: what happens if there not exist file "/etc/hbase/conf/hbase-site.xml"

**Review by javeme — CHANGES_REQUESTED** [review]
just rebase -i master and pick one commit

File: hugegraph-hbase/src/main/java/com/baidu/hugegraph/backend/store/hbase/HbaseSessions.java (hunk 2 above)
Comment (tmljob) [reply hugegraph-hbase/src/main/java/com/baidu/hugegraph/backend/store/hbase/HbaseSessions.java:139]: The program runs normally.If not modified,program read the file from hbase-common-2.0.6.jar, it does not exist actually but prograpm runs normally.

```
  public static Configuration addHbaseResources(Configuration conf) {
    conf.addResource("hbase-default.xml");
    conf.addResource("hbase-site.xml");

    checkDefaultsVersion(conf);
    return conf;
  }
```

**Review by javeme — APPROVED** [review]

**Review by Linary — APPROVED** [review]

**Review by zhoney — APPROVED** [review]

**Pull request merged**